android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java

```diff
@@ -44,16 +43,13 @@ public ImageViewWithUrl(Context context) {
 class FastImageViewManager extends SimpleViewManager<ImageViewWithUrl> implements ProgressListener {
 
     private static final String REACT_CLASS = "FastImageView";
-
     private static final String REACT_ON_LOAD_START_EVENT = "onFastImageLoadStart";
-
     private static final String REACT_ON_PROGRESS_EVENT = "onFastImageProgress";
-
+    private static final String REACT_ON_ERROR_EVENT = "onFastImageError";
+    private static final String REACT_ON_LOAD_EVENT = "onFastImageLoad";
+    private static final String REACT_ON_LOAD_END_EVENT = "onFastImageLoadEnd";
     private static final Drawable TRANSPARENT_DRAWABLE = new ColorDrawable(Color.TRANSPARENT);
-
-    private static final Map<String, List<ImageViewWithUrl>> VIEWS_FOR_URLS = new WeakHashMap<>();
-
-    private static RequestManager requestManager = null;
+    private static final Map<String, List<ImageViewWithUrl>> VIEWS_FOR_URLS = new HashMap<>();
```

Comment:
An't you leaking the whole view ? seem's like a big memory leak !!

Reply:
Yes you are right, removing static references. Next commit will be amended

Reply:
Yes you are right. Switching to private next commit .

Reply:
Added GlideConfiguration for next push